app/modules/video-loader.js

```diff
@@ -25,7 +27,7 @@ export default class VideoLoader {
 	 * @returns {boolean}
 	 */
 	isProvider(name) {
-		return Boolean(this.data.provider.toLowerCase().match(name));
+		return this.data.provider.toLowerCase() === name;
```

Comment:
Please revert it, there are providers like `ooyala/funimation`. It may be good idea to add a comment explaining that.